about.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>The Boonie BootCamp</title>
+    <!-- Link Lato font from Google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;1,400;1,700&display=swap"
+      rel="stylesheet"
+    />
+    <!-- font-family: 'Lato', sans-serif; -->
+    <!-- Link file for the cocogoose font -->
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <!-- font-family: 'COCOGOOSE', sans-serif; -->
+    <!-- Link to general style file -->
+    <link rel="stylesheet" href="css/style.css" />
+    <!-- Link to MOBILE style file -->
+    <link
+      rel="stylesheet"
+      media="screen and (max-width: 768px)"
+      href="css/mobile.css"
+    />
+    <!-- Link to DESKTOP style file -->
+    <link
+      rel="stylesheet"
+      media="screen and (min-width: 768px)"
+      href="css/desktop.css"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="hamb-toggle"></div>
+      <div class="menu-box">
+        <nav class="secondary-menu-box desktop-pad-version">
+          <ul>
+            <li>
+              <a href="#"
+                ><img
+                  src="IMG/social/icon-facebook.svg"
+                  alt="small facebook icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="IMG/social/icon-twitter.svg"
+                  alt="small twitter icon"
+              /></a>
+            </li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+          </ul>
+        </nav>
+        <nav class="primary-menu-box desktop-pad-version">
+          <a class="special-a" href="index.html"
+            ><img src="./IMG/organico-blanco-y-negro.png" alt="logo"
+          /></a>
+          <div class="primary-menu-box-content">
```

Comment:
- [ ] **Required**  In the Mobile version , the hamburger background should be same as described in original website.
- Your website 
![Screenshot from 2022-04-14 01-22-50](https://user-images.githubusercontent.com/26767383/163261036-f032be1c-1c87-4bad-994d-364bf95cc1f0.png)


- Original website 
![Screenshot from 2022-04-14 01-30-55](https://user-images.githubusercontent.com/26767383/163261053-72c84e32-c7a8-4245-a727-e6542da22d67.png)

**Requirement**
- It contains a hamburger menu on the left side.
- It's background is transparent or the same color/pattern as the top section background.